src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -181,6 +181,7 @@ sylius_inventory:
         inventory_unit:
             classes:
                 model: %sylius.model.order_item_unit.class%
+                repository: Sylius\Bundle\CoreBundle\Doctrine\ORM\OrderItemUnitRepository
```

Comment:
I don't think it is as simple as that. Since the CoreBundle configuration does not have the inventory_unit resource section node, it will not override it by just loading the `orm.xml` file which contains the same key for the classname `sylius.repository.inventory_unit.class`